physics/GFS_radiation_surface.F90

```diff
@@ -109,6 +112,14 @@ subroutine GFS_radiation_surface_run (                            &
       ! Intialize intent(out) variables
       sfcalb = 0.0
 
```

Comment:
This is good.  However, I can't see the full code here.  Is fraco open water fraction?  If so, fracl+fraco+fraci=1 right?

Reply:
Thanks for checking. You are correct, `fracl+fraco+fraci=1` for the fractional mask case. Not that in the git diff view, you can expand the sections that are hidden by clicking on the arrows in the blue boxes between the line numbers on the left. I'll merge this into my branch/PR.